ibridges/interactive.py

```diff
@@ -0,0 +1,58 @@
+"""Interactive authentication with iRODS server.
+"""
+
+import json
+import os
+from pathlib import Path
+from typing import Optional, Union
+from getpass import getpass
+
+from ibridges.irodsconnector.session import Session
+
+DEFAULT_IENV_PATH = Path(os.path.expanduser("~")).joinpath(".irods", "irods_environment.json")
+
+def interactive_auth(password: Optional[str] = None,
+                     irods_env_path: Optional[Union[str, Path]] = DEFAULT_IENV_PATH) -> Session:
```

Comment:
Since we don't support env_path == None
```suggestion
                     irods_env_path: Union[str, Path] = DEFAULT_IENV_PATH) -> Session:
```